packages/server/src/models/shelf.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, PrimaryGeneratedColumn, PrimaryColumn } from 'typeorm';
+
+@Entity()
+export class Shelf {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  description: string;
+
+  @Column()
+  location: string;
+
+  @Column()
+  pictureUrl: string;
+}
+
+@Entity()
+export class ShelfFlowerOrdering {
+  @PrimaryColumn()
+  shelfId: number;
+
+  @Column('simple-array')
+  flowerIds: number[];
```

Comment:
Are they gonna be numbers, or maybe strings? I am just thinking that all the ids should be rather UUIDs rather than numbers

Reply:
Agreed to do it the other way around, making all the IDs auto-generated numbers, including userId.
Generating UUIDs would make sense if we'd share the IDs in any way, but they're completely internal. We use tokens if we need to send things out.